common.mk

```diff
@@ -5,26 +5,33 @@ ifndef common_dir
 endif
 
 build = $(common_dir)/build.sh
+test-openshift = $(common_dir)/test-openshift.sh
 
 ifeq ($(TARGET),rhel7)
 	OS := rhel7
 else
 	OS := centos7
 endif
 
-script_env = \
+build_env = \
 	SKIP_SQUASH=$(SKIP_SQUASH)                      \
-	UPDATE_BASE=$(UPDATE_BASE)                      \
+	UPDATE_BASE=$(UPDATE_BASE)
+
+script_env = \
+	VERSION="$(VERSION)"                            \
 	VERSIONS="$(VERSIONS)"                          \
 	OS=$(OS)                                        \
-	VERSION="$(VERSION)"                            \
```

Comment:
> I'm curious why VERSION was moved above

I was thinking about grouping these two similar variables together:-) Sry, not needed...

> is the VERSION && VERSIONS needed, or is that just leftover

At least one is needed. So what about removing `VERSION`? 
So in case user wants only one version to build, `VERSIONS` variable has to be rewritten. But what about backward compatibility?